ndcli/README.md

```diff
@@ -3,24 +3,66 @@ ndcli
 
 This is ndcli, the CLI client for DIM.
 
-installation
-------------
+Linux (.deb, .rpm)
+------------------
 
-Make sure you have the dimclient installed and then install ndcli.
+Packages are available on [github](https://github.com/1and1/dim).
 
+
+Installation in venv
+--------------------
+Thanks to almueller for providing this.
+
+Create virtualenv
 ```
-pip install -r ./requirements.txt
-pip install ./
+virtualenv -p python3 $HOME/ndcli
```

Comment:
According to https://packaging.python.org/en/latest/guides/installing-using-pip-and-virtual-environments/#creating-a-virtual-environment the commands should be:

```
python3 -m venv dim_env
./dim_env/bin/pip install dnspython
./dim_env/bin/pip install https://github.com/1and1/dim/releases/download/dimclient-1.0.1/dimclient-src-1.0.1.tar.gz
./dim_env/bin/pip install https://github.com/1and1/dim/releases/download/ndcli-5.0.2/ndcli-src-5.0.2.tar.gz
```

Still need to test it.